config/routes.rb

```diff
@@ -3,12 +3,11 @@
     resources :govbox do
       collection do
         get :faq, path: 'casto-kladene-otazky'
-        get :register_step1, path: 'registracia'
-        post :register_step2, path: 'registracia-subjekt'
-        post :register_step3, path: 'registracia-statutar'
-        post :register_step4, path: 'registracia-nastavenie'
-        post :register_step5, path: 'registracia-zabezpecenie'
-        post :register_step6, path: 'registracia-odoslanie'
+        get :register_step1, path: 'registracia-subjekt'
```

Comment:
Ten prvy krok nemenme url, ktovie odkial mame vsade linky.
```suggestion
        get :register_step1, path: 'registracia'
```